### PR TITLE
fix: use filename as default name istead of full path

### PR DESCRIPTION
**Pull request by dkontorovskyy**
### What this does
Use filename as default name instead of the full path

### CODE REVIEW

File: test/lib/index.test.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    'package-json-without-name/package.json',
+  );
+  const lockFileContent: string = await readFixture(
+    'package-json-without-name/package-lock.json',
+  );
+  const absolutePath = path.resolve(
+    __dirname,
+    'lib',
+    'fixtures',
+    'package.json',
```
Comment (orsagie) [inline test/lib/index.test.ts:41]: can you change this to something else to show that this absolutePath is in fact being used properly. (currently both tests have the exact same assert, which makes it hard to see what the difference between both calls is)

File: lib/index.ts [hunk 2]
```diff
@@ -76,7 +76,7 @@ async function buildDepTree(
 
   const manifestFile: ManifestFile = parseManifestFile(manifestFileContents);
   if (!manifestFile.name) {
-    manifestFile.name = defaultManifestFileName;
+    manifestFile.name = path.basename(defaultManifestFileName);
```
Comment (orsagie) [inline lib/index.ts:79]: how is changing the name going to affect existing projects?

File: test/lib/index.test.ts (hunk 1 above)
Comment (dkontorovskyy) [reply test/lib/index.test.ts:64]: The difference is in incoming parameters. That output should be the same. That is why I have same asserts :)

File: lib/index.ts (hunk 2 above)
Comment (dkontorovskyy) [reply lib/index.ts]: Good catch! Fixed

**Comment by dkontorovskyy** [comment]
@orsagie, in yarnv2 for CLI we will be calling `buildDepTree` instead of `buildDepTreeForFiles`, because of that we are gonna be passing default name from CLI. This gonna brake yarnWorkspaces functionality. With this small fix, it will act as it used to be.

**Pull request merged**